docs/meetings/2022/README.md

```diff
@@ -3,3 +3,4 @@
 Presentations from the KYOSS monthly meetings in 2022.
```

Comment:
If we go with the above suggestion of listing *all* presentations in the month-specific pages, perhaps we could change this title to not be specific to "Presentations"...?